models/pom.xml

```
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>transformation-api</artifactId>
+        <groupId>com.connexta.transformation</groupId>
+        <version>0.3.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Transformation Rest Models</name>
```

```suggestion
    <name>Transformation :: API :: Rest Models</name>
```
My thought is that, if the artifact name will have "api" in it then the name of the module should too.